sdk/tables/azure-data-tables/samples/async_samples/sample_update_upsert_merge_entities_async.py

```diff
@@ -186,24 +186,12 @@ async def update_entities(self):
             finally:
                 await table.delete_table()
 
-    async def clean_up(self):
-        from azure.data.tables.aio import TableServiceClient
-
-        tsc = TableServiceClient.from_connection_string(self.connection_string)
-
-        async with tsc:
-            async for table in tsc.list_tables():
-                await tsc.delete_table(table.name)
-
-        print("Cleaned up")
-
 
 async def main():
     sample = TableEntitySamples()
     await sample.create_and_get_entities()
     await sample.list_all_entities()
     await sample.update_entities()
-    await sample.clean_up()
```

Comment:
Why did we remove the cleanup?

Reply:
As I see it call delete_table() in the end of each test, so there's no necessary to have another clean_up() after.